app/features/TransactionSlice.ts

```diff
@@ -330,17 +242,19 @@ export const updateTransactions = createAsyncThunk<
         }
 
         async function updateSubroutine(maxId: bigint) {
-            if (!account || controller.isAborted) {
-                controller.finish();
```

Comment:
Why have you removed the finish statements?

Reply:
Is moved to `extraReducers` in slice in most recent update